### PR TITLE
Feature student registration with PrimeReact

### PR DESCRIPTION
**Pull request by ishtiaqniloy**

### CODE REVIEW

File: pages/registration/application/[application_id].js [hunk 1]
```diff
@@ -1,157 +1,211 @@
...
+            <DataTable
+              value={tableFormattedData}>
+              <Column field="field" header="Registration ID"/>
+              <Column field="value" header="Name"/>
+            </DataTable>
+          </div>
+
+          <Divider />
+
+          <div style={{ margin: "auto", marginBottom: "1em" }}>
```
Comment (haque-lubna) [inline pages/registration/application/[application_id].js:180]: styles should be written in css files?

Comment (ishtiaqniloy) [reply pages/registration/application/[application_id].js:180]: @ishmum123 bhaia, kindly let us know if these should be moved to separate css files

Comment (ishmum123) [reply pages/registration/application/[application_id].js:180]: Let's leave it here for the time being... we should get back to this

Comment (ishtiaqniloy) [reply pages/registration/application/[application_id].js:180]: Okay bhai. In that case, would you merge the PR if agreed by lubna apu

Comment (haque-lubna) [reply pages/registration/application/[application_id].js:180]: okay vaiya

**Review by haque-lubna — APPROVED** [review]

**Review by Shuhin — APPROVED** [review]

**Pull request merged**